react-cra/src/App.test.tsx

```diff
@@ -1,5 +1,6 @@
 import { render, screen } from '@testing-library/react';
 import App from './App';
+import '@testing-library/jest-dom/extend-expect';
```

Comment:
This wasn't required before?

Reply:
Wasn't needed with transpiling in Jest. I think it was magically done in `setupTests.ts` with the `@testing-library/jest-dom` import but that doesn't work with separate ts_project transpiliation.